app/serializers/beach_serializer.rb

```diff
@@ -0,0 +1,53 @@
+# == Schema Information
+#
+# Table name: stores
+#
+#  id                  :bigint           not null, primary key
+#  name                :string
+#  group               :string
+#  street              :string
+#  city                :string
+#  district            :string
+#  country             :string
+#  zip_code            :string
+#  latitude            :float
+#  longitude           :float
+#  capacity            :integer
+#  details             :text
+#  store_type          :integer          default("1"), not null
+#  created_at          :datetime         not null
+#  updated_at          :datetime         not null
+#  lonlat              :geometry         point, 0
+#  state               :integer          default("1")
+#  reason_to_delete    :text
+#  open                :boolean          default("true")
+#  created_by_id       :bigint
+#  updated_by_id       :bigint
+#  from_osm            :boolean          default("false")
+#  original_id         :bigint
+#  source              :string
+#  make_phone_calls    :boolean          default("false")
+#  phone_call_interval :integer          default("60")
+#
+
+class BeachSerializer
+  include FastJsonapi::ObjectSerializer
+  set_key_transform :dash
+  set_type :stores
```

Comment:
for this purpose shouldn't we use type :beaches ?

Reply:
Done